static/app/views/settings/account/apiTokenRow.tsx

```diff
@@ -77,8 +84,14 @@ const Details = styled('div')`
   margin-top: ${space(1)};
 `;
 
-const ScopesWrapper = styled('div')`
+const TokenWrapper = styled('div')`
   flex: 1;
+  margin-right: ${space(1)};
+`;
+
+const ScopesWrapper = styled('div')`
+  flex: 2;
```

Comment:
is this doing something specific for you?

Reply:
It brings the "scopes" closer to the token column in the details. I'll be honest though, I'm not great with CSS. 🙃 If there's a correct way to do this, I'm open to it.

Here's what it looks like with `flex: 1`:

![image](https://github.com/getsentry/sentry/assets/20070360/6dc7119f-780f-44c4-87ab-4bf52063772b)